Beginners/Lesson One/README.md

```diff
@@ -0,0 +1,127 @@
+Hey! So we're going to be looking at Swift from the perspective of learning programming. This means you get to ask awkward questions and I struggle to verbalise concepts that are the programming wquivilent of muscle memory. It'll be fun!
+
+To start of we need to get to a point where we can write any code. We're going to assume that you already have a copy of Xcode 7 installed. With that, let's get started!
+
+Open Xcode and you'll get this Welcome dialogue. 
+
+![Xcode Welcome Screen](img/welcome.png)
+
+We'll be using _playgrounds_ to learn Swift. Click "Get started with a playground", give it a name, and make sure the platform is set to **OS X**. 
+
+![Creating a new playground](img/newplayground.png)
+
+Awesome. Xcode will create an open the playground for you. It'll look something like this. 
+
+![Empty Playground](img/emptyplayground.png)
+
+The large pane on the left is where we're going to write code. On the right is where the results of our code are displayed. You can see that there's already one line of code written for us, `var str = "Hello, playground"`, and the results pane says `"Hello, playground"`. Neat. 
+
+
+Normally for things I work with, the code we are writing gets transformed from something we, as humans, kind-of understand to something a machine understands via a compiler. Let's stick with the idea that it's basically a translator from human to machine. Normally its a big upfront conversion, for example Eigen can take 5 minutes to compile. 
+
+We're going to use Playgrounds, which is more like a back and forth conversation with the compiler, so it's much faster to write code and test things out. These are called REPLs.
+
+What this means is whatever you type will be run by Xcode and the results will be shown on the right. When Xcode is compiling code (whenever you change text), the top bar of the window will change to say "Running" and will show a little spinner. 
+
+Let's start by saying what _isn't_ code. You see the the first line, how it is greyed out? Well that's a comment. Comments are annotations in the code that are not ran by the compiler. We'll be using comments occasionally to skip bits of code, or to remind us what something is.
+
+OK let's write some code, let's try typing some things. 
+
+```swift
+1 + 2
+
+1 / 2
+1.0 / 2.0
+
+"Hello, Artsy"
+```
+
+You can see the results in the results pane:
+
+![Results](img/results.png)
+
+In programming, we want to work with abstractions. So let's make our first variable. We're going to make a simple calculator with a variable. What I'd like to do is define a variable called `a` and `b` then add them together to make `c`. This is a trivial example, but it's about foundations.
```

Comment:
Maybe define what a variable is? Could be useful just to say "a variable holds a value that you can refer to later."